commands/randfuture.js

```diff
@@ -0,0 +1,60 @@
+const factory = require('../lib/factory')
+const logger = require('../lib/logger')
+const moment = require('moment')
+const randSched = require('../lib/random-schedule')
+
+exports.command = 'randfuture <extension> <resources>'
+
+exports.describe = 'starts some chaos at a random point in the next month\nNB: there are no time window contraints, can be anytime in a 24 hour period'
+
+exports.builder = {
+  extension: {
+    description: 'the name of the extension',
+    required: true
+  },
+  resources: {
+    description: 'the resources (see `chaos resgen`)',
+    required: true
+  }
+}
+
+exports.handler = (argv) => {
+  const authenticator = factory.AzureAuthenticator.create()
+  const registry = factory.ExtensionRegistry.create()
+
+  const asyncAuthProvider = argv.accessToken ? Promise.resolve(argv.accessToken) : authenticator.interactive()
+
+  var secondsInterval = randSched.calcRandomSecond(1, 2638000)
+  var scheduledTime = moment().add(secondsInterval, 'seconds')
+
+  var resources = argv.resources.split('/')
```

Comment:
we should try to avoid this parsing here, it's duplicated logic that's subject to change (see notes below for if we need this at all)